x/payments/spec/05_events.md

```diff
@@ -0,0 +1,60 @@
+# Events
+
+The Payment module emits the following events:
+
+## EndBlocker
+
+| Type                            | Attribute Key            | Attribute Value       |
+|---------------------------------|--------------------------|-----------------------|
+| payment_contract_authorisation  | payer-did                | {payer-did}           |
+| payment_contract_authorisation  | payment_contract_id      | {payment_contract_id} |
+| payment_contract_authorisation  | authorised               | {authorised}          |
+| create_subscription             | subscription_id          | {subscription_id}     | 
+| create_subscription             | payment_contract_id      | {payment_contract_id} |
+| create_subscription             | max_periods              | {max_periods}         |
+| create_subscription             | key_period               | {key_period}          |
+| create_subscription             | creator_did              | {creator_did}         |
```

Comment:
These should be under MsgCreateSubscription

Reply:
created